src/site/_data/blogTags.js

```diff
@@ -0,0 +1,373 @@
+module.exports = {
```

Comment:
this whole file seems really awkward. Can we not derive slugs from uppercase tags?

I don't have a strong opinion, I just thought I'd call it out.

Reply:
I'm not sure I'm following, can you expand a bit more on what you were thinking?

Reply:
My point is that every single title => tag mapping could be achieved with something like [slugify](https://www.npmjs.com/package/slugify) (although not in reverse due to capitalization differences). Having said that, it would be particularly awkward to specify tags as e.g. "Case Study", not "case-study", in article code.

So I'm happier but my point is this could be a file which derives slugs from a simpler list or config file, e.g.:

```js

const tags = [
  "3D",
  "Accessibility",
  "Augmented Reality",
  // ...
];

module.exports = Object.fromEntries(tags.map((title) => [slugify(title), {title}]));
```

... but I'm possibly overthinking it.

Reply:
got it. Yeah that's definitely another option. I'm not opposed to switching to something like that... @MichaelSolati wdyt?

Reply:
actually, on second thought, maybe the JSON is the right approach. Sam's later comment mentions wanting to update the page title/description and there's a separate thread where Kayce and Joe wanted to explore the idea of tag synonyms (https://github.com/GoogleChrome/web.dev/issues/1808#issuecomment-548617577).

So maybe we keep the JSON object for now.

Reply:
Yea, the JSON object is important to have data to pass into pages.